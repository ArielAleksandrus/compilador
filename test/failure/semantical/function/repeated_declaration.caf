int soma(int a, car b[]){
	retorne a + b[0];
}
int soma(int a, car c[]){
	retorne a + c[0];
}

programa{}