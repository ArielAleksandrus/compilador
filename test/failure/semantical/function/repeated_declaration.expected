ERROR => Line 4, col 8: Redeclaration of function soma(int, car[]). Previously found in line 1 col 8
